pulp_smash/pulp_smash_internals.ipynb

```diff
@@ -0,0 +1,989 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Pulp-Smash Internals"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Why?\n",
+    "\n",
+    "Get familiar with Pulp-Smash, and get to know how Pulp is tested. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Using Jupyter Notebook\n",
+    "\n",
+    "You can also follow along using a **Jupyter Notebook**.  Assuming you are bringing your own Pulp machine. \n",
+    "\n",
+    "\n",
+    "``` shell\n",
+    "python3 -m venv pulp-smash-internals\n",
+    "source ~/pulp-smash-internals/bin/activate\n",
+    "pip install pulp-smash\n",
+    "pip install jupyter\n",
+    "# Create settings related to the system to be tested\n",
+    "pulp-smash settings create\n",
+    "mkdir code && cd code\n",
+    "git clone https://github.com/PulpQE/pulp-qe-tools.git\n",
+    "cd pulp_smash\n",
+    "jupyter notebook\n",
+    "\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### What is Pulp?\n",
+    "\n",
+    "Pulp is a platform for managing repositories of software packages and making it available to a large numbers of consumers. Pulp can locally mirror all or part of a repository, host your own software packages in repositories, and manage many types of content from multiple sources in one place.\n",
+    "\n",
+    "https://pulpproject.org/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Pulp Versions\n",
+    "\n",
+    "There are currently 2 versions of Pulp being tested by Pulp-Smash.\n",
+    "\n",
+    "* **Pulp2** - As of this writing 2.19.1 is the stable release.\n",
+    "\n",
+    "    [Pulp 2 Releases](https://pulp.plan.io/projects/pulp/wiki/Release_Schedule#Past-Releases)\n",
+    "\n",
+    "\n",
+    "* **Pulp3** - Under development - Release Alpha."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### What is Pulp-Smash?\n",
+    "\n",
+    "Pulp-Smash is functional test suite written in Python, to help test Pulp Project. Pulp-Smash is currently used to test Pulp2, and Pulp3. \n",
+    "\n",
+    "Most of the new tests are for Pulp 3.\n",
```

Comment:
I am not sure how this statement is relevant to someone looking into `pulp-smash`.

Maybe `Most of the active development for tests are in Pulp 3.`?